config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml

```diff
@@ -634,7 +634,7 @@ periodics:
       name: ""
       resources: {}
 - annotations:
-    testgrid-dashboards: google-windows, sig-windows-gce, sig-release-1.18-informing, sig-windows-1.18-release
+    testgrid-dashboards: google-windows, sig-windows-gce, sig-windows-1.18-release
```

Comment:
Was initially thinking we would only remove them for the newer jobs but it looks like several of these have been failing for awhile